config/crd/bases/sagemaker.aws.amazon.com_hostingautoscalingpolicies.yaml

```diff
@@ -29,18 +29,24 @@ spec:
     status: {}
   validation:
     openAPIV3Schema:
-      description: HostingAutoscalingPolicy is the Schema for the HostingAutoscalingPolicy API
+      description: HostingAutoscalingPolicy is the Schema for the HostingAutoscalingPolicy
+        API
       properties:
         apiVersion:
-          description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+          description: 'APIVersion defines the versioned schema of this representation
```

Comment:
why is this PR changing other CRDs

Reply:
I had to update the `go.mod` file because it was referencing older versions that had broken upstream dependencies.